Description: Don't use timestamped gzip for reproducible builds.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: http://gmt.soest.hawaii.edu/issues/759
Applied-Upstream: http://gmt.soest.hawaii.edu/projects/gmt/repository/revisions/14772

--- a/doc/rst/CMakeLists.txt
+++ b/doc/rst/CMakeLists.txt
@@ -111,7 +111,7 @@ if (SPHINX_FOUND)
 			-w "${CMAKE_CURRENT_BINARY_DIR}/man.log"
 			${CMAKE_CURRENT_BINARY_DIR}/_source
 			${CMAKE_CURRENT_BINARY_DIR}/man
-			COMMAND ${GZIP} -9 -f ${CMAKE_CURRENT_BINARY_DIR}/man/*.?gmt
+			COMMAND ${GZIP} -9 -n -f ${CMAKE_CURRENT_BINARY_DIR}/man/*.?gmt
 			COMMENT "Building manpages...")
 		add_dependencies (docs_man docs_depends)
 	endif (GZIP)
